cypress/e2e/BiographyTestsFrontEnd/index.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('page title is {string}',(title)=>{
+cy.get(`h1:contains("${title}")`).should('be.visible').and('include.text',title)
+})
+
+And('the individuals title is {string}',(chiefCommunicationTeam)=>{
+cy.get(`div:contains("${chiefCommunicationTeam}")`).should('be.visible').and('include.text',chiefCommunicationTeam)
+})
+
+And('the following organizations are listed for this individual',()=>{
+cy.get('.profile-org').should('be.visible')
+cy.get('.profile-org').should('be.visible')
+})
+
+And('individuals office is at {string}',(nciCompus)=>{
+cy.get("div[class='profile-panel-content'] p").contains(nciCompus).should('be.visible');
+})
+
+And('contact phone number is {string}',(phoneNumber)=>{
+cy.get('.profile-contact-info').contains(phoneNumber).should('be.visible');
+})
+
+And('contact email is {string}',(contactEmail)=>{
+cy.get(`a:contains("${contactEmail}")`).should('be.visible')
```

Comment:
locator too generic - needs to be more specific ( hint here email link has a parent, so make a scope within that parent)